ember-cli-build.js

```diff
@@ -2,10 +2,27 @@
 /* global require, module */
 require('dotenv').config();
 
-var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+const BroccoliMergeTrees = require('broccoli-merge-trees');
+const transpiler = require('broccoli-babel-transpiler');
+const Funnel = require('broccoli-funnel');
 
 module.exports = function(defaults) {
-    var app = new EmberApp(defaults, {
+    const workerDestDir = '/assets/workers';
+
+    const workers = transpiler(
+            new Funnel('app/workers', {
+            include: ['*.js'],
+            destDir: workerDestDir
+        })
+    );
+
+    const workerDeps = new Funnel('bower_components', {
```

Comment:
I believe so, but it would only apply to the account creation page. Seems like an acceptable tradeoff.